clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/ServerStoreProxy.java

```diff
@@ -31,10 +31,12 @@
 
   private final String cacheId;
   private final EhcacheClientEntity entity;
+  private final ServerStoreMessageFactory messageFactory;
 
-  public ServerStoreProxy(String cacheId, EhcacheClientEntity entity) {
+  public ServerStoreProxy(String cacheId, EhcacheClientEntity entity, ServerStoreMessageFactory messageFactory) {
```

Comment:
Like the comment on `EhcacheClientEntity`, I don't see the point of passing this in -- unless we're going to use a *single* instance for all stores under a cache manager (which, based on the implementation of `ServerStoreMessageFactory`, is presently not possible).